web/src/page_params.ts

```diff
@@ -37,6 +37,10 @@ export const page_params: {
     translation_data: Record<string, string>;
     zulip_plan_is_not_limited: boolean;
     muted_users: {id: number; timestamp: number}[];
+    annual_price: number;
+    monthly_price: number;
+    percent_off: number;
+    seat_count: number;
```

Comment:
The `page_params` for the billing page is different from the homepage one, albeit both being `page_params`. Maybe we can create a separate `page_params` for billing only.
https://github.com/zulip/zulip/blob/9e4529c48bee7da41b32888d5f055f7c070b9078/corporate/views/upgrade.py#L274-L280

Reply:
I don't think I fully understand what's going on here 😅
I mean doesen't this context is being used for rendering the `corporate/upgrade.html` template only?
Can you elaborate a little that why and how were we using same `page_params` if they were different from the start!

Reply:
Right. `page_params.ts` is also used in other places where these properties are not available. So adding `seat_count` and etc to `page_params` would not be correct in general.

Reply:
We can avoid importing or changing `page_params.ts`, and in a prep commit, access `$("#page-params").remove().data("params")` directly, and extract the needed fields to be passed to `initialize` in `upgrade.js`.

Reply:
Oh, okay that makes sense to me, but we also need `page_params.seat_count` data in `helper.js` also will it make more sense to make a new separate `page_params` for billing?

Reply:
Yeah, that sounds reasonable for now. Eventually, we will get rid of `page_params`.

Reply:
Yeah that sounds good I will try to make a seperate `page_params` in billing directory for now!

Reply:
@PIG208 So I was trying to access `$("#page_params")` in a newly created `page_params.ts` in billing directory but of course, I couldn't access it because for now, we are removing `#page_params` element in the main `page_params.ts`, any suggestions, should I change main `page_params.ts` to not remove the `#page_params` element from the DOM?

Reply:
I think you shouldn't need to change the main `page_params.ts`. We only remove `#page_params` once upon initialization. You are just replacing the main `page_params.ts` with a local version for billing. It should function exactly the same way.

Reply:
Alright! so made a new commit introducing new `page_params` as well as adjusted the testing code for it!